swagger.json

```diff
@@ -514,6 +577,80 @@
           "example": "https://gravatar.com/avatar/aed61baf1e9256ed7d70e2cbbfcba9aa?s=400&d=robohash&r=x"
         }
       }
+    }, "update": {
```

Comment:
Why are you adding an 'update' request documentation to the _/auth/login_ route?